docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
+
+
+== About Walkthroughs
```

Comment:
Is this information only or is this an introduction to all walkthroughs?

Reply:
good question, no.. its not an introduction to all walkthroughs, that needs to be addressed beforehand. This is about walkthroughs in the context of "now, you want to write your own walkthrough"